deploy/template.yaml

```diff
@@ -3373,6 +3389,32 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
+  YotiCallbackStateMachineFailedCanary:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub "${AWS::StackName}-YotiCallbackStateMachineFailedCanary"
+      AlarmDescription: !Sub "Errors returned from the PSendYotiLetterStateMachine"
+      ActionsEnabled: true
+      AlarmActions:
+        - !ImportValue platform-alarm-topic-critical-alert
+      OKActions:
+        - !ImportValue platform-alarm-topic-critical-alert
+      InsufficientDataActions: []
+      MetricName: ExecutionsFailed
+      Dimensions:
+        - Name: StateMachineArn
+          Value: !Sub "arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:F2F-YotiCallback-stepfunction-${AWS::StackName}"
+        - Name: Alias
+          Value: 'live'
+      Namespace: AWS/States
+      Statistic: Sum
+      Unit: Count
+      Period: 60
+      EvaluationPeriods: 2
```

Comment:
Just to confirm is this intentionally set to 2 e.g. due to normal baseline errors (the other step function was set to 1)?